### PR TITLE
Add log-activity to pin PATCH method

### PR DESCRIPTION
**Pull request by parnurzeal**

### CODE REVIEW

File: src/services/pin/pin-model.js [hunk 1]
```diff
@@ -7,28 +7,25 @@ const Schema = mongoose.Schema;
...
-// disable _id field
-{ _id: false });
+});
 
 const commentSchema = new Schema({
   created_time: { type: Date, required: true, default: Date.now },
   detail: { type: String, required: true },
   mentions: [{ type: Schema.Types.ObjectId, ref: 'User' }],
-  photos: [Schema.Types.ObjectId],
+  photos: [String],
```
Comment (supasate) [inline src/services/pin/pin-model.js:16]: Not link to Photo model?

File: src/services/pin/pin-model.js [hunk 2]
```diff
@@ -7,28 +7,25 @@ const Schema = mongoose.Schema;
...
   tags: [String],
   updated_time: { type: Date, required: true, default: Date.now },
-  videos: [Schema.Types.ObjectId],
+  videos: [String],
```
Comment (supasate) [inline src/services/pin/pin-model.js:19]: Not link to Video model?

**Comment by parnurzeal** [comment]
Changed them to be the same as pin's photo field. Seems we are already
using it as String.

**Comment by supasate** [comment]
LGTM

**Comment by supasate** [comment]
Close #170 
Close #171

**Pull request merged**